banzai/context.py

```diff
@@ -32,22 +32,22 @@ def telescope_passes(self, telescope):
 
 
 class PipelineContext(object):
-    def __init__(self, args, allowed_instrument_criteria):
-        self.processed_path = args.processed_path
-        self.raw_path = args.raw_path
-        self.post_to_archive = args.post_to_archive
-        self.post_to_elasticsearch = args.post_to_elasticsearch
-        self.elasticsearch_url = args.elasticsearch_url
-        self.fpack = args.fpack
-        self.rlevel = args.rlevel
-        self.db_address = args.db_address
-        self.log_level = args.log_level
-        self.preview_mode = args.preview_mode
-        self.filename = args.filename
-        self.max_preview_tries = args.max_preview_tries
-        self.elasticsearch_doc_type = args.elasticsearch_doc_type
-        self.elasticsearch_qc_index = args.elasticsearch_qc_index
-        self.no_bpm = args.no_bpm
-        self.allowed_instrument_criteria = allowed_instrument_criteria
-
-        logs.set_log_level(self.log_level)
+    def __init__(self, command_line_args, allowed_instrument_criteria, processed_path='/archive/engineering/',
+                 post_to_archive=False, fpack=True, rlevel=91, db_address='mysql://cmccully:password@localhost/test',
+                 log_level='INFO', preview_mode=False, max_tries=5, post_to_elasticsearch=False,
+                 elasticsearch_url='http://elasticsearch.lco.gtn:9200', elasticsearch_doc_type='qc',
+                 elasticsearch_qc_index = 'banzai_qc', **kwargs):
```

Comment:
I would say yes, because the defaults here will be useful for testing other stages. The defaults in main are what the user sees. In principle those do not have to match.